### ISSUE TITLE
timeout. not yet replicated

### ISSUE THREAD

**Issue by skram** [rarepress/rarepress#5]
Running some tests and when trying to create multiple, consistently running into timeout's and retrys. Example log

create symlink fs/ipfs/bafybeiakstrsbmkficap2dxvmd7nlw2atfhigbenuptoqalakat7rrl6du
create symlink fs/ipfs/bafkreido6b4j7kvuku5oo5xpmxic3kj3vza36tpq4etyledy7yvqhh4opy
Swarm listening on /ip4/127.0.0.1/tcp/4002/p2p/QmWgwqA7qJwvTHLV8qvkuq3oh2cn9Y25mrUqGGaNxQQwZZ
Swarm listening on /ip4/192.168.1.12/tcp/4002/p2p/QmWgwqA7qJwvTHLV8qvkuq3oh2cn9Y25mrUqGGaNxQQwZZ
Swarm listening on /ip4/127.0.0.1/tcp/4003/ws/p2p/QmWgwqA7qJwvTHLV8qvkuq3oh2cn9Y25mrUqGGaNxQQwZZ
adding to ipfs...
success
adding to ipfs...
timeout. not yet replicated: bafkreido6b4j7kvuku5oo5xpmxic3kj3vza36tpq4etyledy7yvqhh4opy
retrying...
timeout. not yet replicated: bafkreido6b4j7kvuku5oo5xpmxic3kj3vza36tpq4etyledy7yvqhh4opy
retrying...
timeout. not yet replicated: bafkreido6b4j7kvuku5oo5xpmxic3kj3vza36tpq4etyledy7yvqhh4opy
retrying...

and continues endlessly..Any insight?

**Comment by josephf** [comment]
@skram any chance you figured out what's going on? I have the same issue..

**Comment by skram** [comment]
@josephf Nope, i did seem to make it slightly better by removing the cache that's created in `fs/ipfs/` directory before running.

**Comment by kingchenc** [comment]
> @josephf Nope, i did seem to make it slightly better by removing the cache that's created in `fs/ipfs/` directory before running.

This is not working for me, its just pure luck for me with the timeout

95% timeouts, 5% works, why? 

edit: seems if the internet connection not so good we get a timeout, but i have 1 gbit/s connection, so idk whats that bug is

**Comment by Bloodday** [comment]
Same thing happening here, lots of timeouts just by uploading one file

**Comment by mozzillation** [comment]
any update on this?

**Comment by w601sxs** [comment]
This is still happening. Any updates, or any recommendations from the team on how to work around this?

**Comment by viresse5** [comment]
Have the same issue.  Any update?

**Comment by koxon** [comment]
I have the same issue. I had no luck with this yet. Any info on how to make this work?

**Comment by tiagozip** [comment]
Also happening to me. is this project dead?
